docs/DeveloperGuide.md

```diff
@@ -527,76 +525,200 @@ Use case ends
     Use case ends.
 
 - 3a. The given index is invalid.<br>
-   - 3a1. McGymmy shows an error message.
-
-    Use case resumes at step 2.
+   - 3a1. McGymmy shows an error message.<br>
+   Use case resumes at step 2.
 
 **Use case: UC04 Help**
 
 **MSS**
 1. User requests help.
-2. McGymmy shows all commands and examples of command usages.
-
+2. McGymmy shows all commands, and the usage of each command.<br>
 Use case ends.
 
 **Extensions**
 No extensions.
```

Comment:
Not sure if we should have a break line before this

Reply:
No break will result in them being on the same line

Reply:
Actually u can try to view in the rich text version and see

Reply:
Wait nvm edited